### ISSUE TITLE
Episode not released!

### ISSUE THREAD

**Issue by ConnieSCG** [pystardust/ani-cli#1228]
**Metadata (please complete the following information)**
Version: [e.g. 2.0.2]
OS: Arch Linux 6.5.7
Shell: bash 5.1.16
Anime: Several Anime

**Describe the bug**
Episode not released!

**Steps To Reproduce**

1. Run `ani-cli [whatever anime in here]`
2. choose series
3. select episode

**Expected behavior**
Downloading should work

**Additional context**

**Comment by nicelifehaver** [comment]
currently having the exact same issue, also on arch

**Comment by port19x** [comment]
Can't reproduce. Update ani-cli via your package manager or via `ani-cli -U`

**Comment by yawnbo** [comment]
Same issue on zsh on mac

**Comment by unrealapex** [comment]
It looked like the issue revolved itself for me. I will report back with possible fixes soon.

**Comment by MysteriousLounger** [comment]
I've encountered this issue in the past few days. Specifically, certain _newly released_ episodes say that they are not released, even though they are available on the source website. To reproduce, try the following:
`ani-cli -d -r 3 mahou shoujo ni akogarete`
For me, this gives "episode not released"; However, episodes 1 and 2 of this show work as expected.
I'm on version 4.7.3, running on ubuntu 23.10

**Comment by port19x** [comment]
> I've encountered this issue in the past few days. Specifically, certain _newly released_ episodes say that they are not released, even though they are available on the source website. To reproduce, try the following: `ani-cli -d -r 3 mahou shoujo ni akogarete` For me, this gives "episode not released"; However, episodes 1 and 2 of this show work as expected. I'm on version 4.7.3, running on ubuntu 23.10

That's something to look into, I'll try to reproduce it

**Comment by Kaukov** [comment]
I encountered the same issue last night. Wanted to watch the new episode of Solo Leveling (Ore dake Level Up na Ken). The newest episode (5) comes up in the list, but when selecting it, it prints out the error "Episode not released!".

The other episodes of the same anime load without issues though.

I watched that episode from a website because of this. And now, more than 12h later, it's still happening with ani-cli.

ani-cli v4.7.0 running on Gentoo with kernel 6.7.3.
